src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/CertificateAssetDownloader.cs

```diff
@@ -64,6 +64,7 @@ internal static SafeX509CrlHandle DownloadCrl(string uri, ref TimeSpan remaining
                 }
             }
 
+            Interop.Crypto.ErrClearError(); 
```

Comment:
I just fixed this one locally :smile:.  It needs to be inserted between lines 59 and 61.  (so it cleans up errors from DecodeX509Crl when PemReadBioX509Crl succeeds)

Reply:
Yep, you are right, I will take your change instead 😀 (just to be clear: this one is to give the others a feeling for the tactical changes...)